docs/examples.md

```diff
@@ -5,18 +5,84 @@ PyScript can be found [on PyScript.com](https://pyscript.com/@examples).
 
 The examples are (links take you to the code):
 
+### Simple:
+
 * [Hello world](https://pyscript.com/@examples/hello-world/latest)
+  * uses included datetime module. No additional packages.
 * [A simple clock](https://pyscript.com/@examples/simple-clock/latest)
-* [Simple slider panel](https://pyscript.com/@examples/simple-panel/latest)
-* [Streaming data panel](https://pyscript.com/@examples/streaming-in-panel/latest)
+  * Not found
 * [WebGL Icosahedron](https://pyscript.com/@examples/webgl-icosahedron/latest)
-* [KMeans in a panel](https://pyscript.com/@examples/kmeans-in-panel/latest)
-* [New York Taxi panel (WebGL)](https://pyscript.com/@examples/nyc-taxi-panel-deckgl/latest)
+  * uses very old version of [three.js](https://threejs.org/) directly imported in index.html. (not as a module in toml) 
 * [Pandas dataframe fun](https://pyscript.com/@examples/pandas/latest)
+  * uses [pandas](https://pandas.pydata.org/) from pypi
+  * pyodide
 * [Matplotlib example](https://pyscript.com/@examples/matplotlib/latest)
+  * uses [matplotlib](https://matplotlib.org/)
+  * not clear how numpy is being loaded as it is not referenced.
+  * pyodide
+* [Todo](https://pyscript.com/@examples/todo-app/latest)
+  * uses pyweb. No additional packages.
+  * pyodide
+* [Tic Tac Toe](https://pyscript.com/@examples/tic-tac-toe/latest)
+  * uses pyweb. No additional packages.
+  * pyodide
+* [Pyscript Jokes](https://pyscript.com/@examples/pyscript-jokes/latest)
+  * uses pyweb and [pyjokes](https://pyjok.es/)
+  * pyodide
+* [D3 visualization](https://pyscript.com/@examples/d3-visualization/latest)
+  * uses [d3]()
```

Comment:
missing link?

Reply:
added link.
Do you think these examples should also be updated ? - beyond my expertise sorry.

Reply:
I am not sure we changed so much up to the point we need updates but if sme example is badly broken I'd rather try to fix it or understand why is that, if that makes sense.

Reply:
Four primary issues you might want to revisit:

- Icosahedron example
  - because it uses the older style of three.js directly imported in index.html.
  - version 165 is now 27 releases behind today's version.
  - soon after 165 they changed to modules. So you cannot use this style of import unless its limited to 165 or earlier.
  - Now should be instantiated as a module in toml
  - This bug reported here: https://github.com/pyscript/examples/issues/15 which includes links to potential solutions
  - Changing this means users could use three.js into the forseeable future.

- Simple slider panel
  - because it does not run (for me)
 
- KMeans in a panel
  - because it shows an error in the overlay - meaning the screen can't be seen or used by anyone running it
  - error is to do with threadpools (on my machine)

- matplot example
  - because it imports numpy but numpy is not mentioned in the toml or index.html file.
  - I assume this is because it is directly embedded in the pyodide being referenced.
  - From memory there is one more example which also does not import numpy but uses it. (I did not make a note - sorry :( )
  - There are other examples which do formally import numpy in the toml and are therefore better examples.
    -  You can see them referenced in this PR by searching the page for numpy
  
I will update this PR to remove the 'simple clock' you have since removed so it doesn't reappear if this PR is accepted.

Reply:
If the examples are fixed - I would be very pleased to update this PR to meet your satisfaction.

Reply:
I'll try to have a look today ... but about numpy, it's true that explicit is always better, but Pyodide modules resolution takes into account dependencies too ... if matplotlib depends on numpy, it gets imported. This is not instantly clear though, so I agree if used in the PyScript code, that module should be part of the dependencies graph/config.

Reply:
I see everything ifne bt slightly darker with latest Three.JS ... do you think this woul dbe a great example though? module imported via TOML and via `pyscript.js_modules` and there's no `pyodide.ffi` needed anymore

Reply:
P.S. forgot the link, now edited ... the thing is, I don't know anything about ThreeJS so apologies if I did something wrong ... but actually I haven't changed the code at all

Reply:
would this Matplotlib update work for you? https://pyscript.com/@agiammarchi/matplotlib-copy/latest?files=README.md,index.html,pyscript.toml,main.py

Reply:
last, but not least, would this KMeans demo work for you? https://pyscript.com/@agiammarchi/kmeans-in-panel-copy/latest?files=README.md,pyscript.toml,index.html (it takes a while to show the page but it works now)

Reply:
btw, this last one is hard to maintain due the plethora of foreign `<script>` dependencies hard to track and tackle via the bokeh import ... we should keep all those versions updated but I wonder if that's worth our time ... at least seeing something, even if not latest, seems to be desired though, which is what happens now.

Reply:
I'll check these later toay - Thanks for doing this Andrea. I really appreciate it.

wrt the Kmeans demo
 - its a nice demo of kmeans but I agree that this kind of demo with a fragile environment may not be the best demo to include in the examples. 
 - Maybe we could designate a "fragile" category, or we should replace it (and what its trying to show users) with another example which demonstrates the same things but using a less fragile example.
 - This could be a task added to the community to maybe encourage them to make examples that illustrate various parts of pyscript that you decide should be illustrated ??
   - you/someone would need to identify aspects that should be illuminated by examples. Then people could make those.

Reply:
imho we should improve somehow expectations, at least on PyScript .com side ... with that demo all I see is the inability for ourselves, or our users, to pin a config related dependency on both the Python *and* the browser side. I am sure it wouldn't be trivial to solve this issue, and we have a way to pin dependencies, but unfortunately that way doesn't necessarily play well with Pyodide registry (not their fault neither). Having a way to keep examples updated is one thing we should consider **but** having a way to help users keeping their own examples updated seem like a great goal too.

I will discuss this with @ntoll once he's back, but it's clear to me current strategy is not necessarily the best we could offer.

Reply:
my notes:


- Icosahderon
  - I will submit PR to this to also update the lighting so it looks better.
  - I hope that having this example included will prompt more three.js examples
  
- simple panel
  - I followed your link and it just shows a slider but nothing else.
  - Ahhh I think I understand. The slider IS the demo.
    - I guess I expected more than just a slider with a feedback value

- matplotlib
  - looks great
 
- kmeans
  - clicking on the plot does not filter but generates a bug. 
  - The onscreen instruction indicates this should work
  - error is:
  ```
  PyodideTask exception was never retrieved
future: <PyodideTask finished name='Task-41' coro=<write() done,
   defined at /lib/python3.12/site-packages/panel/io/pyodide.py:489> exception=Error:
   NumberFormatter(p1053).text_align given invalid value: {"value":"left"}>
```

I will update my PR

Reply:
PR updated. Also added new PR to fix icosahedron example on pyscript-examples:
  - https://github.com/pyscript/examples/pull/23